sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -185,33 +179,115 @@
     "    type=AssetTypes.URI_FOLDER,\n",
     "    path=\"test_dataset/\",\n",
     ")"
-   ]
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "To create data input from TabularDataset created using V1 sdk, specify the `type` as `AssetTypes.MLTABLE`, `mode` as `InputOutputModes.DIRECT` and `path` in the following format `azureml:<tabulardataset_name>:<version>`."
```

Comment:
Hi Nikolay, this format azureml:<tabulardataset_name>:<version> is also in the notebooks of other verticals. Would it be better if I reframe it as : "To create data input from TabularDataset created using V1 sdk, set the type to AssetTypes.MLTABLE, mode to InputOutputModes.DIRECT and path to the following format azureml:<tabulardataset_name> or azureml:<tabulardataset_name>:<version> (in case we want to use specific version). ".
Please let me know.

Reply:
Yes, we can use this wording.